src/components/space/ActionsChart.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import { BarChart, CartesianGrid, XAxis, YAxis, Tooltip, Bar } from 'recharts';
+import { getActionsByDay, formatActions } from '../../utils/api';
+import spaceData from '../../data/sample.json';
+
+function ActionsChart() {
+  const actionsByDay = formatActions(getActionsByDay(spaceData));
+
+  return (
+    <Container>
+      <Typography variant="h6" style={{ textAlign: 'center' }}>
+        Actions by day
+      </Typography>
+      <BarChart
+        width={620}
+        height={450}
```

Comment:
The height should be fixed, but the `width` should be responsive. I think you can use `100%`.

Reply:
Fixed. Recharts provides a ResponsiveContainer to hold chart components. The BarChart component itself does not take non-integer values for width/height.